tests/docstrings_examples/DocTest.res

```diff
@@ -9,24 +9,19 @@ type example = {
   docstrings: array<string>,
 }
 
-type error =
-  | ReScriptError(string)
-  | RuntimeError({rescript: string, js: string, error: string})
-
-let bscBin = Path.join(["cli", "bsc"])
-
-let parsed = Util.parseArgs({
-  args: Process.argv->Array.sliceToEnd(~start=2),
-  options: dict{"ignore-runtime-tests": {Util.type_: "string"}},
-})
-
-let ignoreRuntimeTests = switch parsed.values->Dict.get("ignore-runtime-tests") {
-| Some(v) =>
-  v
-  ->String.split(",")
-  ->Array.map(s => s->String.trim)
-| None => []
-}
+// Ignore some tests not supported by node v18
+let ignoreRuntimeTests = [
```

Comment:
Bonus points: detect node version and run tests accordingly. As these tests actually work with Node 22.

Reply:
Done.